test/index.js

```diff
@@ -54,10 +59,16 @@ test("@import", function(t) {
   compareFixtures(t, "empty-and-useless", "should work with empty files")
 
   compareFixtures(t, "transform", "should support transform", {
-    path: importsDir,
     transform: require("css-whitespace"),
   })
 
+  compareFixtures(t, "plugins", "should support plugins", {
+    plugins: [
+      require("postcss-nested"),
+      require("postcss-bem-linter"),
```

Comment:
instead of adding dev deps (real plugins) we should code & use a stupid plugin with more stupid fixtures